python/examples/property-selection.py

```diff
@@ -0,0 +1,121 @@
+
+# We can use a subset of these names to define a selection.
+# In this case, only properties matching the labels in this selection will
+# be used by rascaline (here, only `l=0` components will be used)
+selection = Labels(
+    names=["l"],
+    values=np.array([[0]]),
+)
+selected_descriptor = calculator.compute(frames, selected_properties=selection)
+
+selected_descriptor.keys_to_samples("species_center")
+selected_descriptor.keys_to_properties(["species_neighbor_1", "species_neighbor_2"])
+
+properties = selected_descriptor.block().properties
+# get an array([0]) as a list of angular properties
+print(f"we have the following angular components: {np.unique(properties['l'])}")
+
+# The previous selection method uses the same selection for all blocks.
+# If you can to use different selection for different blocks, you should
+# use a `TensorMap` to create your selection
+
+selected_descriptor = calculator.compute(frames, selected_properties=selection)
+descriptor_for_comparison = calculator.compute(
+    frames, selected_properties=selected_descriptor
+)
+
+# The descriptor had 180 properties stored in the first block,
+# the selected_descriptor had 36. So descriptor_for_comparison
+# will also have 36 properties.
+print("shape of first block initially:", {descriptor.block(0).values.shape})
+print("shape of first block of reference:", {selected_descriptor.block(0).values.shape})
+print(
+    "shape of first block after selection:",
+    {descriptor_for_comparison.block(0).values.shape},
+)
+
+
+# The `TensorMap` format allows us to select features within
+# each block, and then construct a general matrix of features. We
+# can select the most significant features using FPS, which selects
+# features based on the  distance between them. The following algorithm
+# selects the 10 most important features in each block, then constructs
+# a new descriptor based on them, and calculates the final matrix of
+# features for it.
+
+# choose features:
+def fps_feature_selection(descriptor, n_to_select):
+    blocks = []
+    for _, block in descriptor:
+        # create a separate FPS selector for each block
+        fps = FPS(n_to_select=n_to_select)
+        mask = fps.fit(block.values).get_support()
+        selected_properties = block.properties[mask]
+        # put the selected features in a format rascaline can use
+        blocks.append(
+            TensorBlock(
+                # values, samples and component carry no information here
+                values=np.empty((1, len(selected_properties))),
+                samples=Labels.single(),
+                components=[],
+                properties=selected_properties,
+            )
+        )
+
+    return TensorMap(descriptor.keys, blocks)
+
+
+# We get `TensorMap` in the results of this function. The only thing that
+# is important in it is the properties, so we can create samples and
+# components empty altogether. In the case of values, we only need
+# form matching with len of properties.
```

Comment:
I would move this comment (only the properties are important, we leave the rest empty) inside the function above